package.py

```diff
@@ -353,11 +340,13 @@ def setup_miniupnpc(self) -> None:
 
         build_dir = self.__storage.build_directory
         os.chdir(build_dir)
-        url = f'https://github.com/mrx23dot/miniupnp/releases/download/miniupnpd_2_2_24/{miniupnc}'
-        zip_filename = Downloader.download(url)
-        zip_path = build_dir / zip_filename
+        zip_path = build_dir / miniupnc 
```

Comment:
how did this pass linting @kelsos @yabirgb? This fails linting in develop right now (I am fixing in my PR). It has a trailing whitespace.

Reply:
I recall running the linter but probably is my fault chaning something. Will check this weekend how to configure the hooks. The CI doesn't trigger the linter for this file. Thanks

Reply:
Yes, make lint will fix errors, but this file is not actually watched by the detection change mechanism, so unless there are also backend changes, it can miss it. I should probably change it so that you can also specify single files to the job action